osu.Game/Database/ArchiveModelManager.cs

```diff
@@ -97,11 +98,11 @@ public void Import(params string[] paths)
 
                 try
                 {
-                    notification.Text = $"Importing ({i} of {paths.Length})\n{Path.GetFileName(path)}";
+                    notification.Text = $"Importing ({success} of {paths.Length})\n{Path.GetFileName(path)}";

```

Comment:
While here do we want to address the "Importing 0 of 123" for readability?

Reply:
yeah i'll change that too